README.md

```diff
@@ -292,16 +292,96 @@ which saves you from defining a lot prefixes.
 
 Add to your project `build.gradle` file (Latest version is [![DeeplinkDispatch version](https://badge.fury.io/gh/airbnb%2FDeepLinkDispatch.svg)](https://badge.fury.io/gh/airbnb%2FDeepLinkDispatch)
 ): 
+```groovy
+dependencies {
+  implementation 'com.airbnb:deeplinkdispatch:x.x.x'
+}
+```
+
+DeeplinkDispatch supports three ways to run the annotation processor dependin on which one you choose
+the setup is slightly different.
+
+### KSP
+
+When using Kotlin we strongly suggest to use KSP as it can bring major speed improvements.
+We ran into issues when enabling KSP on a project while the Kapt plugin was still applied.
+If you want to both use Kapt and KSP processors on the same project be careful and expect
```

Comment:
I wonder how much of this was just caused by the no java sources issues - Eli talked a bit more to the KSP team, and running both kapt and ksp should be fine, so long as no kapt processor depends on the ksp generated sources

Reply:
Yeah, I wanted to remove this actually, thx for pointing it out.